package.json

```
@@ -12,25 +12,26 @@
   ],
   "scripts": {
     "format": "prettier --write \"**/*.{js,md}\"",
+    "format:check": "prettier --check \"**/*.{js,md}\"",
     "lint": "eslint",
```

```suggestion
    "lint": "eslint .",
```

Seems like eslint@6 just prints the help options when you don't specify a path:
https://app.circleci.com/pipelines/github/salesforce/eslint-config-lwc/75/workflows/10ee9089-33c8-4f5e-9336-975ba1157ead/jobs/75

But eslint@7 is fine without it:
https://app.circleci.com/pipelines/github/salesforce/eslint-config-lwc/75/workflows/10ee9089-33c8-4f5e-9336-975ba1157ead/jobs/76

---

Good catch @ekashida. I think we need to run `yarn lint` first and then install a different eslint version to run the tests.